tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go

```diff
@@ -437,6 +549,16 @@ func getErrorMetricsValue(val *prommodel.MetricFamily) float64 {
 				}
 			}
 		}
+	case "keda_scaledjob_errors_total":
```

Comment:
This isn't consistent with prometheus test case. There the metric name is `keda_scaled_job_errors`

Reply:
Thanks for the quick feedback! 
Cause scaledObject error has suffix _total, I used that one without full context.

I updated with one more commit.
https://github.com/kedacore/keda/pull/4913/commits/ff50b0f8f103f3ca8594c81d27e62d6d6c209cc5

Reply:
I meant with the _ in the middle `scaledjob` & `scaled_job`

Reply:
Could you elaborate it more? I couldn't fully get to the point for this :)

Reply:
yeah, you are right, sorry
I've checked more in depth and the metric you created for otel is `keda.scaledjob.errors` 
![image](https://github.com/kedacore/keda/assets/36899226/de36d66b-b023-43e2-b0ea-bbb9ac3e1831)

so the metric that you can expect in the test is `keda_scaledjob_errors` (the testing process replaces `.` with `_`).

Following the current approach for other metrics, your otel metric should be `keda.scaledjob.error.totals` and the otel e2e test case will receive the metric `keda_scaledjob_error_totals`

Reply:
Thanks. Now I think I get to it more.
<img width="898" alt="image" src="https://github.com/kedacore/keda/assets/38376255/036c47a2-64ee-41b8-a4b7-3e3f33ee4485">
I amended my last commit like the screenshot.

BTW, if that is, should scaledobject be changed together?
[keda/tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go](https://github.com/kedacore/keda/blob/d3751e925e96726c6b1a4152d3f1140e7f8e01a0/tests/sequential/opentelemetry_metrics/opentelemetry_metrics_test.go#L379)

Reply:
OMG!, 
I don't understand anything now 🤦 Sorry for the huge confusion I'm generating :( 
We should use the same approach as the metric that already exist. Replicating current naming but for the ScaledJob, all the rest was correct :(
And sorry again for the confusion

Reply:
No worries. It's all my pleasure. The more we discussed, the better I am getting to keda :)
BTW, 
`We should use the same approach as the metric that already exists`
<= What does this mean? 
Do I need to update from now? How?

Reply:
The metric should be named:
- otel: `keda.scaledjob.errors`
- prometheus: `scaled_job_errors_total` (this is a bit different because this will be the final value after [this refactor](https://github.com/kedacore/keda/pull/5174))

Then you have to update the e2e test cases for matching these names

Reply:
Hi, then current changes seems ok to move on.

**otel**

- https://github.com/kedacore/keda/pull/4913/files#diff-1b98f9b287bb8783cd25e2877e1897ecfca1e2a73c6852339679014d211c51ecR90
- https://github.com/kedacore/keda/pull/4913/files#diff-d696b11fc649daa9d9f875b32e54c5766a78622138a6458f9a4b1386501e14d3R477

**prometheus**

- Cause the PR hasn't been merged yet, current change can move forward.